Library/Formula/dmd.rb

```diff
@@ -59,7 +59,7 @@ def install
     # linked into opt by the time this build runs:
     conf.write <<-EOS.undent
         [Environment]
-        DFLAGS=-I#{include}/d2 -L-L#{lib}
+        DFLAGS=-I#{include}/dlang/dmd -L-L#{lib}
```

Comment:
This possibly seems worth bumping the revision if we're actively moving files around?